src/aelfrice/retrieval.py

```
@@ -392,6 +392,16 @@
 META_POSTERIOR_TEMPERATURE_STATIC_DEFAULT: Final[float] = 0.5
 # Sub-posterior decay — 30d, matching the rest of the #480 family.
 META_POSTERIOR_TEMPERATURE_POSTERIOR_DECAY_SECONDS: Final[int] = 30 * 24 * 3600
+# Default-OFF feature flag for #758 adaptive-learning delivery.
+# Distinct from :data:`ENV_USE_GAMMA_POSTERIOR_TEMPERATURE` (#796): that
+# flag gates whether the γ-rerank uses ``T`` at all; this flag gates
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `γ` (GREEK SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF003)